source/api/galaxy/build_dataset_yaml.py

```diff
@@ -8,17 +8,27 @@
 
 def build_specs(args):
     specs = {
-        args.dataset_name: {
-            "format": args.format,
-            "params": {}
+        "definitions": {
+            "datasets": {
+                args.dataset_name: {
+                    "format": args.format,
+                    "params": {}
+                }
+            },
+        },
+        "instructions": {
+            "my_dataset_generation_instruction": {
+                "type": "DatasetGeneration",
+                "datasets": [args.dataset_name],
+                "formats": ["Pickle"]
+            }
         }
     }
 
     if args.metadata_file == "":
-        specs["dataset"]["params"]["path"] = "./"
+        specs["definitions"]["datasets"][args.dataset_name]["params"]["path"] = "."
```

Comment:
is there a reason why "./" became "."? I think it should work regardless, just curious :)

Reply:
No, I was confused for a second whether they meant the same thing, but they are :)
I don't have a particular preference for using either one

Reply:
yes, i was just thinking about concatenating paths, but it should be fine 👍